src/Tickets/Commerce/Gateways/Stripe/Stripe_Elements.php

```diff
@@ -69,4 +75,22 @@ public function card_element_type() {
 		return tribe_get_option( Settings::$option_checkout_element_card_fields );
 	}
 
+
+	/**
+	 * Are Apple/Google Pay wallets enabled?
+	 *
+	 * @since TBD
+	 *
+	 * @return bool
+	 */
+	public function wallets_enabled() {
+		if ( ! defined( 'EVENT_TICKETS_PLUS_FILE' ) ) {
+			return false;
+		}
+
+		$settings = tribe( \TEC\Tickets_Plus\Commerce\Gateways\Stripe\Settings::class );
+		$wallets  = tribe_get_option( $settings::$option_checkout_element_wallets, [] );
```

Comment:
Let's add a `use \TEC\Tickets_Plus\Commerce\Gateways\Stripe\Settings;`

No need to potentially init an object to access it's static property.
```suggestion
		$wallets  = tribe_get_option( Settings::$option_checkout_element_wallets, [] );
```

Reply:
Yes, @dpanta94 I thought of that but I opted not to, because we won't know if ETP will be available. 

Is it ok to use a namespaced class that might not exist? No warnings or errors will be trown?

Reply:
Yes it is! When you reach that part of the code, you have already checked that ETP exists.

Just `use` a non existent class triggers no errors. It will when you actually use it and it doesn't exist.

So when you would reach that line :D

Reply:
Changed... thanks!